app/src/main/kotlin/no/nav/aap/oppgave/plukk/ReserverOppgaveService.kt

```diff
@@ -70,5 +70,22 @@ class ReserverOppgaveService(
         return oppgaverSomSkalReserveres
     }
 
+    fun tildelOppgaver(
+        oppgaver: List<Long>,
+        ident: String,
```

Comment:
```suggestion
        tildelTilIdent: String,
```
? Litt klarere, kanskje?